### PR TITLE
Add placeholder Dockerfile: works!

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by evetion** [comment]
You probably want to add the pixi toml to the Dockerfile?

**Comment by Robinlovelace** [comment]
> You probably want to add the pixi toml to the Dockerfile?

Yes, it is in the pixi-r folder, but seemingly Docker cannot see it.

**Comment by martinfleis** [comment]
See the example here - https://github.com/prefix-dev/pixi-docker?tab=readme-ov-file#usage-with-shell-hook. They are copying pixi.toml to the container first before building the environment.

**Comment by u3ks** [comment]
also you can try  to specify the explicit filename and location for the pixi install:

pixi install --manifest-path ~/myproject/pixi.toml

**Comment by Robinlovelace** [comment]
Update with this: the resulting image, that does not have that much stuff in there, is already 3GB+. Too big!

I plan to try auto-building it to check but I suspect we may need to go back to micromamba docker images which seem really well optimised: https://github.com/mamba-org/micromamba-docker

**Comment by martinfleis** [comment]
@darribas spent a long time trying to slim down `gds_env` and I am not too optimistic that you can get much smaller than this if you want relatively complete R and Python stacks. Probably a bit but not a ton.

One thing that would help is cleaning all the cache as pixi will by default cache stuff I think, so there'll be some duplication in the container.

File: pixi-r/pixi.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[project]
+name = "my_project"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+[pypi-dependencies]
```
[the remark this answers is not in the document]
Comment (Nowosad) [reply pixi-r/pixi.toml:6]: @Robinlovelace -- have you seen Martin's comment?

**Comment by Robinlovelace** [comment]
Going to merge now, let's see what happens..

**Pull request merged**